app/views/juntos_bootstrap/projects/_dashboard_posts.html.slim

```diff
@@ -9,6 +9,7 @@
               = render 'devise/shared/alert'
               =  hidden_field_tag 'anchor', 'posts'
               = f.simple_fields_for :posts, @project.posts.build do |form|
+                = form.input :user_id, as: :hidden, input_html: { value: current_user.id }
```

Comment:
Verdade, vc pode pegar isso no controller de projects, se for o caso. Não entendi bem o motivo dessa alteração.